lib/utils.pm

```diff
@@ -642,6 +643,10 @@ sub noupdatestep_is_applicable {
     return !get_var("UPGRADE") && !get_var("LIVE_UPGRADE");
 }
 
+sub installwithaddonrepos_is_applicable {
```

Comment:
utils is overcrowded already . also currently this function called only in single place - onlinerepos . maybe better just move it there ?

Reply:
Not single place, it is in main_common.pm https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f0acb2d4fcc59393da023db6b96178a4c2fab41b/lib/main_common.pm#L537  . And i moved it from main_common to utils. Or do you think that, there is better place than utils?

Reply:
`installwithaddonrepos_is_applicable` It looks a bit strange to use underscore only partly, but that's not a function created in this commit (old function from 1bacc650b). LGTM.

Reply:
> Not single place, it is in main_common.pm
> 
> [os-autoinst-distri-opensuse/lib/main_common.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f0acb2d4fcc59393da023db6b96178a4c2fab41b/lib/main_common.pm#L537)
> 
> Line 537 in [f0acb2d](/os-autoinst/os-autoinst-distri-opensuse/commit/f0acb2d4fcc59393da023db6b96178a4c2fab41b)
>  if (installwithaddonrepos_is_applicable() && !get_var("LIVECD")) { 
> . And i moved it from main_common to utils. Or do you think that, there is better place than utils?

ah miss that , so let's leave it here than